### PR TITLE
Add Apogee Duet Software Installer

### PR DESCRIPTION
**Pull request by nerab**

### CODE REVIEW

File: Casks/apogee-duet.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'apogee-duet' do
+  version 'June_2015'
+  sha256 '441302c1931ee474e36172dd87ad9b26781a96b3b7ad8e18abe587bebad17ce3'
+
+  url "http://www.apogeedigital.com/drivers/Duet_#{version}.dmg"
+  name 'Apogee Duet Software Installer'
```
Comment (vitorgalvao) [inline Casks/apogee-duet.rb]: `Software Installer` should be removed. Too generic.

**Comment by vitorgalvao** [comment]
Why is this in this repo, though? This looks like ti belongs in the main repo.

**Comment by nerab** [comment]
You mean it should be in [homebrew-cask](https://github.com/caskroom/homebrew-cask)?

**Comment by vitorgalvao** [comment]
Yes. Why submit here?

**Comment by nerab** [comment]
I wasn't sure where it belongs, since my [last contribution](/caskroom/homebrew-versions/pull/1573) was only accepted here ;-) I'll move it there. Thanks!

**Comment by nerab** [comment]
I moved it to [homebrew-cask](https://github.com/caskroom/homebrew-cask/pull/17100)

**Pull request closed without merging**